geode-assembly/src/integrationTest/resources/dependency_classpath.txt

```diff
@@ -87,4 +87,4 @@ lucene-queryparser-6.6.6.jar
 lucene-core-6.6.6.jar
 lucene-queries-6.6.6.jar
 geo-0.7.7.jar
-netty-all-4.1.59.Final.jar
+netty-all-4.1.66.Final.jar
```

Comment:
may need a similar change in geode-assembly/src/integrationTest/resources/assembly_content.txt